webapp/src/hooks/hooks.js

```diff
@@ -35,8 +35,10 @@ export default class Hooks {
                 this.store.dispatch(sendEphemeralPost('You already have a Jira account linked to your Mattermost account. Please use `/jira disconnect` to disconnect.'));
                 return Promise.resolve({});
             }
-            if (getInstalledInstanceType(this.store.getState()) === 'server' && isDesktopApp()) {
-                this.store.dispatch(sendEphemeralPost('Please use your browser to connect to Jira.'));
+
+            if (getInstalledInstanceType(this.store.getState()) === 'server' && isDesktopApp() && !isMinimumDesktopAppVersion(4, 3, 0)) { // eslint-disable-line no-magic-numbers
+                const errMsg = 'The current version of your desktop app does not support connecting to Jira directly. Please connect through your browser or [see here](https://github.com/mattermost/mattermost-plugin-jira) for details on upgrading.';
```

Comment:
Here is the error message we show for when the Jira Server user has desktop app before `v4.3.0`